src/AWS.Deploy.Orchestration/DisplayedResources/DisplayedResourcesHandler.cs

```diff
@@ -16,8 +17,8 @@ public interface IDisplayedResourcesHandler
 
     public class DisplayedResourcesHandler : IDisplayedResourcesHandler
     {
-        private readonly IAWSResourceQueryer _awsResourceQueryer;
-        private readonly IDisplayedResourceCommandFactory _displayedResourcesFactory;
+        internal readonly IAWSResourceQueryer _awsResourceQueryer;
```

Comment:
Why are this fields now `internal`? Seems like we are leaking implementation details.

Reply:
Converted these fields to `readonly` properties and included them in the `IDisplayedResourcesHandler` interface.